.travis.yml

```diff
@@ -0,0 +1,35 @@
+# testing file for Travis
```

Comment:
@AnBucquet could you update the PR body (it says "WIP: add .travis.yml-- which seems done)

Reply:
Done